CONTRIBUTING.md

```diff
@@ -0,0 +1,350 @@
+# Contributing to 3DTilesRendererJS
+
+Thanks for taking the time to consider contributing! We very much appreciate your time and effort. This document outlines the many ways you can contribute to our project, and provides detailed guidance on best practices. We look forward to your help!
+
+## Prerequisites
+
+Before you begin contributing to our project, it'll be a good idea to ensure you've satisfied the below pre-requisites. 
+
+### Developer Certificate of Origin (DCO)
+
+To accept your contributions for any kind of patch, you'll want to:
+1. Understand what a Developer Certificate of Origin is. See [this](https://wiki.linuxfoundation.org/dco) guide.
+2. Read and agree to the [terms](https://developercertificate.org) of the Developer Certificate of Origin.
+3. Remember to add your Sign-Off for each patch contribution you submit to our project via either:
+   1. By using the `-s` flag if using Git. See [these](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s) instructions.
+   2. By including the following text in your patch contributions (i.e. pull requests)
+   ```
+   Signed-off-by: Full Name <email>
+   ``` 
+
+Reviewers reviewing your patch will look for the sign-off before deciding to accept your contribution.
+
+### License
+
+Our project has our licensing terms, including rules governing redistribution, documented in our [LICENSE](LICENSE) file. Please take a look at that file and ensure you understand the terms. This will impact how we, or others, use your contributions.
+
+### Code of Conduct
+
+Our Code of Conduct helps facilitate a positive interaction environment for everyone involved with the team, and provides guidance on what to do if you experience problematic behavior. Read more in our [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md), and make sure you agree to its terms. 
+
+### Governance Model
+
+Our Governance model helps outline our project's decision making and roles-based expectations. Read more in our [GOVERNANCE.md](GOVERNANCE.md). 
```

Comment:
There is no GOVERNANCE.md file included in the PR? I also don't think I've ever seen one in a project.

Reply:
@gkjohnson - see rationale in articles like [this one](https://opensource.guide/leadership-and-governance/#how-do-i-formalize-these-leadership-roles). We do have [recommendations for a GOVERNANCE.md](https://nasa-ammos.github.io/slim/docs/guides/governance/governance-model/) though we're not yet pushing out PRs with this just yet.